lib/Patat/Images/WezTerm.hs

```diff
@@ -27,6 +38,26 @@ data Config = Config deriving (Eq)
 instance A.FromJSON Config where parseJSON _ = return Config
 
 
+--------------------------------------------------------------------------------
+data Pane =
+    Pane { paneSize     :: Size
+         , paneIsActive :: Bool
+         } deriving (Show, Generic)
+
+instance A.FromJSON Pane where
+    parseJSON = A.genericParseJSON $ A.defaultOptions { A.fieldLabelModifier = camelToSnakeCase . drop 4 }
```

Comment:
Thanks a lot!  Just one more suggestion -- sorry for the confusion -- when I asked for inlining these, I was talking specifically about just manually defining the Aeson instances:

```suggestion
    parseJSON = A.withObject "Pane" $ \o -> Pane
        <$> o A..: "size"
        <*> o A..: "is_active"
```

That way we don't need the extra snake-casing code, and it's (IMO) easier to see what sort of JSON we expect to parse.